taca/illumina/NovaSeq_Runs.py

```diff
@@ -17,31 +18,50 @@
 
 logger = logging.getLogger(__name__)
 
-class NovaSeq_Run(HiSeqX_Runs):
-     def __init__(self,  run_dir, samplesheet_folders):
+class NovaSeq_Run(HiSeq_Run):
+    def __init__(self,  run_dir, samplesheet_folders):
         super(NovaSeq_Run, self).__init__( run_dir, samplesheet_folders)
         self._set_sequencer_type()
         self._set_run_type()
 
 
-
     def _set_sequencer_type(self):
         self.sequencer_type = "NovaSeq"
 
     def _set_run_type(self):
         self.run_type = "NGI-RUN"
 
-
-    def _get_samplesheet(self):
+    def _generate_clean_samplesheet(self, ssparser):
```

Comment:
Will this be affected when the SampleSheets are generated by `EPP` and also any reason that `_generate_clean_samplesheet` of `HiSeq_Run` class wouldn't work ?